app/src/main/res/values/strings.xml

```diff
@@ -181,10 +181,13 @@ be temporary, and you can try again later.</li>
     <string name="turbo_mode_disabled_toast">Turbo Mode disabled</string>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
+    <string name="onboarding_turbo_mode_body2">Turbo Mode automatically blocks trackers and ads to help get you where you’re going faster. If there’s a speed bump – like a site that isn’t behaving the way you expect – just turn Turbo Mode off.</string>
```

Comment:
How is this different from the string that's above it? The strings should be named to indicate *where* in the UI they're used.

Are the old strings no longer used? In that case you should remove them.

Reply:
Then again, if these strings are identical and used in the same context (just with a redesigned layout), maybe we should just use the first string.

Reply:
The only difference is using curved apostrophes instead of normal ones.

Reply:
I can remove the old strings when I merge the layout changes, but until then we need them or compilation will break.

Reply:
Why are these strings merging independently of the code? In general, it's better to land strings with the code so localizers can see where their strings will be used.